### PR TITLE
Add ability to optionally use +srv records for MongoDB URIs

### PR DESCRIPTION
**Pull request by mrlynn**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -223,10 +227,13 @@ func main() {
...
-	ctx = context.WithValue(ctx, usernameKey, os.Getenv("TODO_MONGO_USERNAME"))
-	ctx = context.WithValue(ctx, passwordKey, os.Getenv("TODO_MONGO_PASSWORD"))
-	ctx = context.WithValue(ctx, databaseKey, os.Getenv("TODO_MONGO_DATABASE"))
+	ctx = context.WithValue(ctx, hostKey, os.Getenv("MONGODB_HOST"))
+	ctx = context.WithValue(ctx, usernameKey, os.Getenv("MONGODB_USER"))
+	ctx = context.WithValue(ctx, passwordKey, os.Getenv("MONGODB_PASS"))
+	ctx = context.WithValue(ctx, databaseKey, os.Getenv("MONGODB_DATABASE"))
+	ctx = context.WithValue(ctx, useSrvKey, os.Getenv("MONGODB_USESRV"))
+	fmt.Printf("%T", ctx)
+	fmt.Println("Context:", ctx)
```
Comment (wemgl) [inline main.go]: Were these for debugging or intentional? If they’re for debugging can you remove them?

File: main.go [hunk 2]
```diff
@@ -298,12 +305,15 @@ func execCmd(ctx context.Context, db *mongo.Database, cmd command) error {
...
-	uri := fmt.Sprintf(`mongodb://%s:%s@%s/%s`,
+
+	uri := fmt.Sprintf(`mongodb%s://%s:%s@%s/%s`,
+		ctx.Value(useSrvKey).(string),
 		ctx.Value(usernameKey).(string),
 		ctx.Value(passwordKey).(string),
 		ctx.Value(hostKey).(string),
 		ctx.Value(databaseKey).(string),
 	)
+	fmt.Printf("HOST: %s", uri)
```
Comment (wemgl) [inline main.go]: Same question here.

**Review by wemgl — COMMENTED** [review]
Thank you for submitting this PR and helping to improve it so it’s more useful for others who’ve read the blog post. I left a few comments, but otherwise the PR looks good to me.

File: main.go (hunk 1 above)
Comment (mrlynn) [reply main.go]: Yes - will clean those up... also - forgot that I renamed the env variables... not necessary... I will switch those back as well.

**Pull request merged**